### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by wombaugh** [MickaelRigault/simsurvey#25]
Good evening! Importing the current simsurvey version does not find simultarget:

import simsurvey

~/github/simsurvey/simsurvey/__init__.py in <module>()
      4 __version__ = "0.4.0"
      5 
----> 6 from simultarget import *
      7 from simulsurvey import *
      8 from models      import *

ModuleNotFoundError: No module named 'simultarget'

**Comment by ufeindt** [comment]
That's odd, simultarget is right in the same folder as __init__.py.

Did the package install without an error? And did you install from pip or using the github version? (They should be the same though.)

**Comment by wombaugh** [comment]
Yes I know - could there be some trivial bug in simultarget that it actually complains about?

We have tried both the pip and github versions, and they yield the same error (as you predicted). Can you recreate it?

**Comment by ufeindt** [comment]
Well, it doesn't seem to find simultarget in the first place. I previously had problems with setup.py not recognizing all submodules (but it's weird since it gets them all for me).

What version of setuptools is install (if any)?

**Comment by MickaelRigault** [comment]
Hi I guess jakob you use python 3?

try adding a point in front of the packages:

from .simultarget import * from .simulsurvey import * from .models import *

**Comment by MickaelRigault** [comment]
Odd the formating:
```
from .simultarget import *
from .simulsurvey import *
from .models import *
```

**Comment by MickaelRigault** [comment]
from .simultarget import *
from .simulsurvey import *
from .models import *

**Comment by wombaugh** [comment]
Yes, that seems to do it. Is this a general python 3 change? 

One thing that was curious was that when we changed the order of the imports, i.e. starting with models, it seems to have at least started with parsing that one.

**Comment by ufeindt** [comment]
Interesting. Actually python 3 is currently not supported. Did pip actually install it for python 3? I thought it wouldn't list the package for it.

There are definitely some other things that should make it crash in python 3, like xrange or the occasional print statement (which I should probably avoid anyway).

**Comment by MickaelRigault** [comment]
I recently moved all my code from python 2.7-> python 3 so I had to fixe this bug.
another usual bug is that dict.keys() is not a list anymore but a `dict_key` object which has some differences. for instance np.asarray(dict.keys()) will bring trouble. 
To solve it, simply change all the dict.keys() -> list(dist.key()). This will be what you used to have.


Uli, if you only have python 3 installed, pip will install things in python 3
